src/components/expanded-state/ens/InfoRow.tsx

```diff
@@ -93,21 +93,24 @@ export default function InfoRow({
     navigate(Routes.EXPLAIN_SHEET, { type: explainSheetType });
   }, [explainSheetType, navigate]);
 
+  const explainer = explainSheetType ? (
+    <ButtonPressAnimation onPress={handlePressExplain}>
+      <Text color="secondary25" size="16px" weight="bold">
+        􀅵
+      </Text>
+    </ButtonPressAnimation>
+  ) : null;
+
   return (
     <Inline alignHorizontal="justify" horizontalSpace="24px" wrap={false}>
       <Box style={{ minWidth: 60, opacity: show ? 1 : 0 }}>
         <Inset top={isMultiline ? '15px' : '10px'}>
           <Inline space="4px">
             <Text color="secondary60" size="16px" weight="bold">
               {label}
+              {android && <Fragment> {explainer}</Fragment>}
```

Comment:
```suggestion
              {android && <Fragment>{explainer}</Fragment>}
```

Reply:
This space is the most important part 😅. Otherwise I wouldn't bother with the `<Fragment>`